templates/named.conf.options.erb

```diff
@@ -73,7 +77,12 @@ check-names response <%= @check_names_response %>;
 	// If BIND logs error messages about the root key being expired,
 	// you will need to update your keys.  See https://www.isc.org/bind-keys
 	//========================================================================
-	dnssec-validation auto;
+	dnssec-validation yes;
```

Comment:
Can you save these changes for a different review? There is another PR for a similar thing.